tests/deploy/test_scheduler_agent.py

```diff
@@ -251,6 +250,9 @@ def check_with_scheduler(self, scheduler: ResourceScheduler) -> None:
             if status:
                 assert scheduler._state.resource_state[resource].status == status
 
+    def set_parameters(self, fact_result: FactResult) -> None:
+        self.state[Id.parse_id(fact_result.resource_id).resource_str()] = const.ResourceState.deploying
```

Comment:
Why deploying?

Reply:
Just copy pasted it and forgot to change

Reply:
I noticed there is not a specific state that I associate with the fact refresh, maybe available. What do you suggest?
```
    unavailable = "unavailable"  # This state is set by the agent when no handler is available for the resource
    skipped = "skipped"  #
    dry = "dry"
    deployed = "deployed"
    failed = "failed"
    deploying = "deploying"
    available = "available"
    cancelled = "cancelled"  # When a new version is pushed, in progress deploys are cancelled
    undefined = "undefined"  # The state of this resource is unknown at this moment in the orchestration process
    skipped_for_undefined = "skipped_for_undefined"  # This resource depends on an undefined resource
```

Reply:
not do anything I think